src/fetch.js

```diff
@@ -1,15 +1,19 @@
+import userStuff from './index.js';
+
 let fetcher = {
  fetchUser(id) {
     fetch(`https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/travelers/travelers/${id}`)
         .then(response => response.json())
-        .then(data => this.fetchTripsForAUser(data))
+        .then(data => userStuff.setUserData(data))
+        .then(this.fetchTripsForAUser())
         .catch(err => console.log('err', err))
 },
 
- fetchTripsForAUser(userData) {
+ fetchTripsForAUser() {
     fetch('https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/trips/trips')
         .then(response => response.json())
-        .then(data => console.log(data.trips.filter(trips => trips.userID === userData.id)))
+        .then(data => userStuff.setUserTrips(data.trips))
+        // .then(data => console.log(data.trips.filter(trips => trips.userID === userData.id)))
```

Comment:
I like the way you have this set up so both fetch methods are easily accessible in the index.js. I assume you'll add to the fetcher variable when you build out the functionality for booking a trip?

Reply:
Yes, I believe so! There are still a few more fetching data points that I need. I think next is fetching destinations, and using those to calculate price among all trips fro a year.